sigmastate/src/test/scala/sigmastate/utxo/DistributedSigSpecification.scala

```diff
@@ -134,14 +128,14 @@ class DistributedSigSpecification extends SigmaTestingCommons {
     val env = Map("pubkeyA" -> pubkeyAlice, "pubkeyB" -> pubkeyBob, "pubkeyC" -> pubkeyCarol)
     val prop = compile(env, """atLeast(2, Coll(pubkeyA, pubkeyB, pubkeyC))""").asSigmaProp
 
-    val (rBob, aBob) = DLogInteractiveProver.firstMessage()
-    val dlBKnown: Hint = RealCommitment(pubkeyBob, aBob)
+    val bobHints = proverB.generateCommitments(prop.treeWithSegregation, ctx)
```

Comment:
Where "She then puts Bob's randomness and fill his response with zero bits. Thus Alice's signature is not valid. Alice is sending her signature to Bob response with zero bits." mentioned in the description is done?

Reply:
In the prover

Reply:
In the prover

Reply:
It is also better to put as comments in the code.